marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/provenance/dh-provenance.xqy

```diff
@@ -0,0 +1,128 @@
+xquery version "1.0-ml";
+
+module namespace dhps = "http://marklogic.com/data-hub/dh-provenance-services";
```

Comment:
I think this library should have a comment to explain the purpose of it. It's sitting in a directory with another library named "prov-lib.sjs". And that begs the question - what's the purpose of each library? i.e. What purposes should I use dh-provenance.xqy for, and what purposes should I use prov-lib.shs for?